src/router.js

```diff
@@ -71,10 +90,17 @@ const router = createRouter({
   ],
 });
 
-// router.beforeEach(async (to) => {
-//   if (Utils.getStore("user") == null) {
-//     return "/login";
-//   }
-// });
+router.beforeEach((to, from, next) => {
+  const user = JSON.parse(localStorage.getItem("user"));
+  if (user === undefined && to.path !=="/login"){
+    next({path: "/login"});
+  }
+  else if (to.path == "/"){
+    authenticateUser(to, from, next);
+  }
```

Comment:
Can we make sure to update the menu bar store to not display the menu actions if the user is not authenticated?

Reply:
Too Difficult to fix